packages/docs/src/page-configs/styles/typography/examples/blockquote.vue

```diff
@@ -1,12 +1,10 @@
 <template>
   <blockquote class="va-blockquote">
     <p>
-      BQ: Let’s talk about meat fondue recipes and what you need to know
-      first. Meat fondue also known as oil fondue is a method of cooking
-      all kinds.
+      {{ $t('typography.examples.blockQuote.content') }}
     </p>
     <p>
-      <i>— Mister Lebowski</i>
```

Comment:
Though we actually translate stuff like this in [vuestic-admin](https://github.com/epicmaxco/vuestic-admin), there it's used as an example/boilerplate of proper application.